src/github-api.js

```diff
@@ -322,6 +324,7 @@ function githubCreateRepo() {
   .then(() => {
     $('#new-repo-name').val("");
     $('#new-repo-desc').val("");
+    $('#new-repo-type').val("public");
     showAlert(`Successfully create new repository ${repo}`);
   })
   .catch((err) => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/21dcc489c3f7718bd929727b9d75c3cf1a977953/src/github-api.js#L330"></span>‘err’ is defined but never used.</td>
    </tr>
  </tbody>
</table>